deploy-manage/security.md

```diff
@@ -75,102 +75,162 @@ $$$maintaining-audit-trail$$$
 
-An Elastic implementation comprises many moving parts: {es} nodes forming the cluster, {kib} instances, additional stack components such as Logstash and Beats, and various clients and integrations communicating with your deployment.
+An Elastic implementation comprises many moving parts: {{es}} nodes forming the cluster, {{kib}} instances, additional stack components such as Logstash and Beats, and various clients and integrations communicating with your deployment.
 
 To keep your data secured, Elastic offers comprehensive security features that:
 - Prevent unauthorized access to your deployment
 - Encrypt communications between components
 - Protect data at rest
 - Secure sensitive settings and saved objects
 
-Security requirements and capabilities vary by deployment. Features may be managed automatically by Elastic, require configuration, or must be fully self-managed. Refer to [Security by deployment type](#security-by-deployment-type) for details.
+:::{note}
+The availability and configurability of security features vary by deployment type. Refer to [Security by deployment type](#security-features-by-deployment-type) for a comparison table.
+:::
 
-::::{tip}
-See the [Deployment overview](/deploy-manage/deploy.md) to understand your options for deploying Elastic.
-::::
+## Security topics
 
-### Security by deployment type
+The documentation is organized into four main areas.
 
-Security features have one of these statuses across deployment types:
+:::{note}
+Throughout the documentation, you'll see deployment type indicators that show which content applies to specific deployment types. Focus on sections tagged with your deployment type and look for subsections specifically addressing your deployment model.
+:::
 
-| Status | Description |
-|--------|-------------|
-| **Managed** | Handled automatically by Elastic with no user configuration needed |
-| **Configurable** | Built-in feature that needs your configuration (like IP filters or passwords) |
-| **Self-managed** | Infrastructure-level security you implement and maintain |
-| **N/A** | Not available for this deployment type |
+### 1. Secure your hosting environment
+
+The [security of your hosting environment](security/secure-hosting-environment.md) forms the foundation of your overall security posture. This section covers environment-specific security controls:
 
-#### Communication security
+- [**Elastic Cloud Hosted and Serverless**](security/secure-your-elastic-cloud-organization.md): Organization-level SSO, role-based access control, and cloud API keys
+- [**Elastic Cloud Enterprise**](security/secure-your-elastic-cloud-enterprise-installation.md): TLS certificates, role-based access control, and cloud API keys
+- [**Elastic Cloud on Kubernetes**](security/secure-your-eck-installation.md): Kubernetes-specific security controls
+- [**Self-managed environments**](security/manually-configure-security-in-self-managed-cluster.md): TLS certificates, HTTPS configuration
```

Comment:
```suggestion
- [**Elastic Cloud Hosted and Serverless**](security/secure-your-elastic-cloud-organization.md)
- [**Elastic Cloud Enterprise**](security/secure-your-elastic-cloud-enterprise-installation.md)
- [**Elastic Cloud on Kubernetes**](security/secure-your-eck-installation.md)
- [**Self-managed environments**](security/manually-configure-security-in-self-managed-cluster.md)
```
Wondering if it's even useful to list those features here to avoid giving the impression that this is all they talk about.

Reply:
agree it's unnecessary here 👍